src/elf.rs

```diff
@@ -157,6 +157,9 @@ where
             elf::header::EM_AARCH64 => Machine::Arm64,
             elf::header::EM_386 => Machine::X86,
             elf::header::EM_X86_64 => Machine::X86_64,
+            elf::header::EM_MIPS | elf::header::EM_MIPS_RS3_LE | elf::header::EM_MIPS_X => {
```

Comment:
MIPS-X is not the same as MIPS.

Reply:
And on second thoughts, please delete the `EM_MIPS_RS3_LE` too unless you have an immediate need for it (which is unlikely).